src/theme/MDXContent/index.js

```diff
@@ -24,7 +24,12 @@ export default function MDXContentWrapper(props) {
 
   const legacy = _.some(_.initial(breadcrumbs), item => item.customProps?.legacy)
   const outdated = !legacy && _.some(_.initial(breadcrumbs), item => item.customProps?.outdated)
-  const offerings = _.find(breadcrumbs, item => item.customProps?.offerings)
+  let offerings; 
+  if (breadcrumbs) {
+    const currentPage = breadcrumbs.slice(-1)
+    const parents = breadcrumbs.slice(0, -1)
+    offerings = _.find(currentPage.map(x => Array.isArray(x.customProps?.offerings)).includes(true) ? currentPage : parents, item => item.customProps?.offerings)
+  }
```

Comment:
```suggestion
 const offerings = _.findLast(breadcrumbs, item => item.customProps?.offerings)
```

I think this might be a bit simpler ;) Also your version seems to only defer to the immediate parent? Would it work when a page overriding the `offerings` is nested deeper?